content/applications/finance/accounting/customer_invoices/deferred_revenues.rst

```diff
@@ -1,195 +1,206 @@
+- Entry 1 dated on the 30th November:
 
-Automate the Deferred Revenues
-==============================
+  - Line 1: Expense account -1200 -600 = **-1800** (we cancel the total of both invoices)
+  - Line 2: Expense account 1100 + 550 = **1650** (we recognize 11/12 of invoice A and 11/12 of invoice B)
+  - Line 3: Deferred account 1800 - 1650 = **150** (amount that has yet to be deferred later on)
 
-When you create or edit an account of which the type is either *Current Liabilities* or *Non-current
-Liabilities*, you can configure it to defer the revenues that are credited on it automatically.
+- Entry 2 dated on the 1st December, the reversal of the previous entry.
 
-You have three choices for the **Automate Deferred Revenue** field:
+In December
+~~~~~~~~~~~
 
-#. **No:** this is the default value. Nothing happens.
-#. **Create in draft:** whenever a transaction is posted on the account, a draft *Deferred Revenues
-   entry* is created, but not validated. You must first fill out the form in
-   :menuselection:`Accounting --> Accounting --> Deferred Revenues`.
-#. **Create and validate:** you must also select a Deferred Revenue Model (see:
-   `Deferred Revenue Models`_). Whenever a transaction is posted on the account, a *Deferred
-   Revenues entry* is created and immediately validated.
+There is no need to generate entries in December. Indeed, if we do the computation for December,
+we will have an amount of 0 to be deferred.
 
-.. image:: deferred_revenues/deferred_revenues09.png
-   :align: center
-   :alt: Automate Deferred Revenue on an account in Odoo Accounting
+In total
+~~~~~~~~
+
+If we aggregate everything, we would have:
+
+- invoice A and invoice B
+- 2 entries (one for the deferral and one for the reversal) for each month from January to November
+
+Therefore, at the end of December, invoices A and B are fully recognized as income,
+only once thanks to the reversal mechanism.
+
+Why would you need the "Manually & Grouped" mode?
+-------------------------------------------------
+
+Suppose you have **1000** invoices.
+
+In the *On validation* mode, you would have:
+1000 invoices × 13 deferral entries (reversal of the original invoices + 12 months) = **13000** deferral entries
 
-.. tip::
-   You can, for example, select this account as the default **Income Account** of a product to fully
-   automate its sale. (see: `Choose a different Income Account for specific products`_).
+In the *Manually & grouped* mode, we do not depend on the number of entries
+as they are all aggregated into one, but only on the periods, so you would have:
+2 entries (deferral + reversal) × 11 months = **22** deferral entries
 
-.. seealso::
-   * :doc:`../get_started/chart_of_accounts`
-   * `Odoo Academy: Deferred Revenues (Recognition) <https://www.odoo.com/r/EWO>`_
+Therefore *Manually & Grouped* mode might be useful when you have lots of deferred revenues
+and you want to reduce the number of journal items created by the deferrals.
```

Comment:
```suggestion
  Therefore *Manually & Grouped* mode is useful when there are lots of deferred revenues and you
  want to reduce the number of journal items created by the deferrals.
```
In my opinion, leaving this "you" is okay because we're talking about if this is the user's specific desire instead of talking about a general example.